torch/_torch_docs.py

```diff
@@ -2845,6 +2845,34 @@ def merge_dicts(*dicts):
         tensor([False,  True,  False,  True,  False])
 """)
 
+add_docstr(torch.isposinf,
+           r"""
+isposinf(input, *, out=None) -> Tensor
+Tests if each element of :attr:`input` is positive infinity or not.
+Args:
+    {out}
+Example::
+    >>> a = torch.tensor([-float('inf'), float('inf'), 1.2])
+    >>> a
+    tensor([  -inf,    inf, 1.2000])
+    >>> torch.isposinf(a)
+    tensor([False,  True, False])
+""".format(**common_args))
+
+add_docstr(torch.isneginf,
+           r"""
+isneginf(input, *, out=None) -> Tensor
+Tests if each element of :attr:`input` is negative infinity or not.
+Args:
+    {out}
+Example::
+    >>> a = torch.tensor([-float('inf'), float('inf'), 1.2])
+    >>> a
```

Comment:
Since you're constructing the tensor `a` it's OK not to print it again

Reply:
addressed.